websites/G/G2G/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "283757164923715587",
+		"name": "Mattis."
+	},
+	"service": "G2G",
+	"description": {
+		"en": "G2G is a world-leading digital marketplace, founded by serious technopreneurs."
+	},
+	"url": "www.g2g.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/U0FQhL3.png",
```

Comment:
Logo probs shouldn't have this white border 
![image](https://user-images.githubusercontent.com/34792838/220175841-ec5ebe50-3b05-4ba5-934f-77a96971cfe8.png)

Reply:
took the logo of a reviewer. I will fix it.

Reply:
Ok so thats a github glitch. You can try download the logo.

Reply:
Nvm, my bad